packages/build-info/src/list-frameworks.ts

```diff
@@ -0,0 +1,7 @@
+import type { Framework } from './frameworks/framework.js'
+
+export const listFrameworks = async ({ projectDir }: { projectDir: string }): Promise<Framework[]> => {
+  // FIXME(serhalp) implement
```

Comment:
I looked into this a little bit and noticed that all references to this function actually seem to want to know what the primary detected framework is instead. Instead of reimplementing this function, we might be able to leverage existing detection functions instead.